src/tests/ui/anonymous-experiences/request.spec.ts

```diff
@@ -129,6 +129,39 @@ test.describe('Run query', () => {
     expect(page.getByText('"Microsoft Viva"')).toBeDefined();
   })
 
+  test('Tests query parameter addition for chained query parameters', async () => {
+    const queryInputField = page.locator('[aria-label="Query sample input"]');
+    await queryInputField.click();
+    await page.evaluate(() => document.fonts.ready);
+    await queryInputField.fill('');
+    await queryInputField.fill('https://graph.microsoft.com/v1.0/user');
+    await queryInputField.press('Tab');
+    await queryInputField.fill('https://graph.microsoft.com/v1.0/users?sel');
+    await queryInputField.press('Tab');
+    await queryInputField.press('Tab');
+    await queryInputField.fill('https://graph.microsoft.com/v1.0/users?$select=id&sel');
+    await queryInputField.press('Tab');
+    await queryInputField.fill('https://graph.microsoft.com/v1.0/users?$select=id&$select=d');
+    await queryInputField.press('Tab');
+    await queryInputField.press('Tab');
+    const queryContent = await page.locator('[aria-label="Query sample input"]').inputValue();
+    expect(queryContent).toBe('https://graph.microsoft.com/v1.0/users?$select=id&$select=deletedDateTime');
```

Comment:
We can improve this here by making sure we:
- can do a chained $ to different parameters e.g $select=x&$filter=y
- can not do a chained $ to the same parameter e.g $select=x&$select=u

Reply:
True. Chaining the same parameter is wrong. Let me make adjustments to cover this case